test/benchmark_http_client_test.cc

```diff
@@ -26,6 +26,26 @@ using namespace testing;
 
 namespace Nighthawk {
 
+namespace BenchmarkClientTestHelperFunctions {
+// Default function for request generator when the content doesn't matter.
```

Comment:
nit: please leave an empty line after the namespace.

Reply:
Done.